llm_bench/python/llm_bench_utils/ov_utils.py

```diff
@@ -271,6 +271,34 @@ def create_ldm_super_resolution_model(model_path, device, **kwargs):
     return ov_model, from_pretrained_time
 
 
+def create_speech_2txt_model(model_path, device, **kwargs):
+    """Create speech generation model.
+
+    - model_path: can be model_path or IR path
+    - device: can be CPU
+    - model_type:
+    """
+    default_model_type = DEFAULT_MODEL_CLASSES[kwargs['use_case']]
+    model_type = kwargs.get('model_type', default_model_type)
+    model_class = OV_MODEL_CLASSES_MAPPING.get(model_type, OV_MODEL_CLASSES_MAPPING[default_model_type])
+    model_path = Path(model_path)
+    model_path_existed = model_path.exists()
+    # load model
+    if not model_path_existed:
+        raise RuntimeError(f'==Failure ==: model path:{model_path} does not exist')
+    else:
+        start = time.perf_counter()
+        ov_model = model_class.from_pretrained(
```

Comment:
Why do we have optimum here? it is supposed to be the the whisper pipeline form GenAI

Reply:
@andrei-kochin both will be supported.

Reply:
Yes.